src/main/java/org/dependencytrack/persistence/jdbi/mapping/OrganizationalContactMapper.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of Dependency-Track.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) OWASP Foundation. All Rights Reserved.
+ */
+package org.dependencytrack.persistence.jdbi.mapping;
+
+import org.dependencytrack.model.OrganizationalContact;
+import org.dependencytrack.persistence.converter.OrganizationalContactsJsonConverter;
+import org.jdbi.v3.core.mapper.ColumnMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrganizationalContactMapper implements ColumnMapper<List<OrganizationalContact>> {
+
+    @Override
+    public List<OrganizationalContact> map(ResultSet r, int columnNumber, StatementContext ctx) throws SQLException {
+        if (r.getBytes(columnNumber) == null) {
+            return new ArrayList<>();
+        }
```

Comment:
I think you can pass the column value directly to the converter. It already handles `null` values: 
https://github.com/DependencyTrack/hyades-apiserver/blob/68f786b699946c2badf231b567aba2c53ea7b84d/src/main/java/org/dependencytrack/persistence/converter/AbstractJsonConverter.java#L75-L77

Also note how the converter returns `null` for `null` inputs, while this mapper would return an empty list, yielding a different behavior. Better to follow what the converter is doing.